packages/block-library/src/embed/edit.native.js

```diff
@@ -236,10 +236,6 @@ const EmbedEdit = ( props ) => {
 		<>
 			{ showEmbedPlaceholder ? (
 				<>
-					<EmbedControls
```

Comment:
Even though the `Edit Button` functionality is being removed, I still kept the `setIsEditingURL` behavior. I will utilize the PR associated with these changes to verify that it is still necessary and functions as expected.